### PR TITLE
Merge spaCy into Development - integrated with core

### PR DESCRIPTION
**Pull request by lastmansleeping**

### CODE REVIEW

File: etk/core.py [hunk 1]
```diff
@@ -546,16 +582,28 @@ def _relevant_text_from_context(text_or_tokens, results, field_name):
...
                         else:
                             new_end = end + 5
+
+                        # result['context']['text'] = ' '.join(
+                        #     text_or_tokens[new_start:new_end])
+                        # result['context']['tokens_left'] = text_or_tokens[
+                        #     new_start:start]
+                        # result['context'][
+                        #     'tokens_right'] = text_or_tokens[end:new_end]
+
```
Comment (saggu) [inline etk/core.py]: why is this commented out code here? Makes me wonder how are you guys resolving conflicts?

File: etk/core.py [hunk 2]
```diff
@@ -546,16 +582,28 @@ def _relevant_text_from_context(text_or_tokens, results, field_name):
...
+                                'utf-8'),
                             relevant_text,
                             ' '.join(text_or_tokens[end:new_end]).encode('utf-8'))
-                        result['context']['tokens_left'] = text_or_tokens[new_start:start]
-                        result['context']['tokens_right'] = text_or_tokens[end:new_end]
+                        result['context']['tokens_left'] = text_or_tokens[
+                            new_start:start]
+                        result['context'][
+                            'tokens_right'] = text_or_tokens[end:new_end]
+
```
Comment (saggu) [inline etk/core.py]: changes like these are everywhere, just a rearrangement of code snippets, please revert to the original and add your changes only

File: etk/unit_tests/test_extractions_using_spacy.py [hunk 3]
```diff
@@ -68,18 +67,33 @@ def test_extraction_from_age_spacy(self):
...
-            extracted_social_media_handles = spacy_social_media_extractor.extract(
-                nlp_doc, self.c.matchers['social_media'])
+    #         extracted_addresses = [address['value']
+    #                                for address in extracted_addresses]
 
-            #print extracted_social_media_handles
+    #         # print extracted_addresses
 
 if __name__ == '__main__':
     unittest.main()
```
Comment (saggu) [inline etk/unit_tests/test_extractions_using_spacy.py]: Take a look at test cases using dictionary for example, add unit tests to run spacy by calling c.process(doc) as well. 
Use the actual extraction_config

File: etk/core.py (hunk 1 above)
Comment (lastmansleeping) [inline etk/core.py]: Because there was a conflict and I was using it as reference. I'll remove it.

File: etk/core.py (hunk 2 above)
Comment (lastmansleeping) [inline etk/core.py]: Alright.

File: etk/unit_tests/test_extractions_using_spacy.py (hunk 3 above)
Comment (lastmansleeping) [inline etk/unit_tests/test_extractions_using_spacy.py]: Do you want both ways or just c.process(doc) now?

Comment (saggu) [inline etk/unit_tests/test_extractions_using_spacy.py]: both ways

Comment (lastmansleeping) [inline etk/unit_tests/test_extractions_using_spacy.py]: This is done. You can merge now.

**Comment by saggu** [comment]
Take a look at this, core.py - https://github.com/usc-isi-i2/etk/pull/36/files?diff=split

There are changes all over the file, you made changes related to only spacy right?
Please re commit this.

**Pull request closed without merging**